lib/spack/docs/index.rst

```diff
@@ -42,7 +42,6 @@ Table of Contents
 -----------------
 
 .. toctree::
-   :maxdepth: 2
```

Comment:
It was recommended as a fix for one of the indexing issues.  I believe it's also the default.  It didn't change the expansion depth when I looked at the output.